src/Impostor.Server/Net/Inner/Objects/GameManager/Logic/GameLogicComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Impostor.Api.Net.Inner;
+using Impostor.Api.Net.Inner.Objects;
+
+namespace Impostor.Server.Net.Inner.Objects.GameManager.Logic;
+
+internal abstract class GameLogicComponent
+{
+    public virtual void OnPlayerDisconnect(IInnerPlayerControl pc)
```

Comment:
Is this needed?

Reply:
I just copied what the base game did. Might be okay to remove this one.